src/charts/brush.js

```diff
@@ -60,55 +60,45 @@ export default function Brush (_container) {
     cache.chartBrush.selectAll(".brush-rect")
       .attr("height", config.chartHeight)
 
-    moveBrush()
+    setBrush()
   }
 
-  function getDataExtent () {
+  function getDataExtentUnderBrush () {
```

Comment:
This was wrongly getting the data extent instead of the actual brush extent. I'm not sure what was the reason.